src/date-picker/date-picker.jsx

```diff
@@ -43,6 +43,11 @@ const DatePicker = React.createClass({
     disableYearSelection: React.PropTypes.bool,
 
     /**
+     * Used to change the first day of week.
+     */
+    firstDayOfWeek: React.PropTypes.number,
```

Comment:
I think it would be good to add something link:
`The first day of the week drastically varies between Saturday, Sunday, or Monday (or even Friday) between different locales`